test/e2e/data/infrastructure/cluster-template-docker.yaml

```diff
@@ -48,7 +48,8 @@ metadata:
   name:  "${CLUSTER_NAME}-control-plane"
 spec:
   template:
-    spec: {}
+    spec:
+      customImage: kindest/node:${KIND_IMAGE_VERSION}
```

Comment:
I'm curious, why do we need to set `customImage`?

Reply:
There is no kind image for all patch releases of K8S, I decided to explicitly set it here so we can use more RKE2 versions.